src/compiler/compiler.ts

```diff
@@ -14,14 +14,15 @@ import ts from 'typescript';
 
 /**
  * Generate a Stencil compiler instance
- * @param config a Stencil configuration to apply to the compiler instance
+ * @param userConfig a user-provided Stencil configuration to apply to the compiler instance
  * @returns a new instance of a Stencil compiler
+ * @public
```

Comment:
Unrelated - I'd like to start annotating publicly exposed functions in some way to flag if we create a breaking change, `@public` isn't _quite_ right here